src/ui/preview/PreviewAction.js

```diff
@@ -75,8 +75,13 @@ export default class PreviewAction {
         PreviewAction.isVisible = true;
     }
 
-    appliesTo(objectPath) {
-        return !PreviewAction.isVisible && !this._isNavigatedObject(objectPath);
+    appliesTo(objectPath, view = {}) {
```

Comment:
I don't think we should default view to `{}`. It should always be provided by the platform, right?

Reply:
it does not work for  `/src/ui/mixins/context-menu-gesture.js`
https://github.com/nasa/openmct/pull/3770/files#diff-67c705006942b6dfa025e97264c78f9a24b8d845d5b31b6d877a8dca91d65540R39